src/modules/privacy-idea-administration/privacy-idea-administration.service.ts

```diff
@@ -198,4 +203,99 @@ export class PrivacyIdeaAdministrationService {
             }
         }
     }
+
+    private async verifyTokenStatus(serial: string, token: string): Promise<TokenVerificationResponse> {
+        const endpoint: string = `/token?serial=${serial}`;
+        const baseUrl: string = process.env['PI_BASE_URL'] ?? 'http://localhost:5000';
+        const url: string = `${baseUrl}${endpoint}`;
+        const headers: { Authorization: string } = {
```

Comment:
Why is PI_BASE_URL taken from the environment-variables and not our ConfigService? 
Wouldn't need to have a default here as well, because the default can be in our well-structured config.json

Also, I'd recommend using the standard `URL` class to construct the url, instead of string concatenation

Reply:
Adjusted with Config